homeassistant/components/intent/timers.py

```diff
@@ -711,6 +744,10 @@ class StartTimerIntentHandler(intent.IntentHandler):
     slot_schema = {
         vol.Required(vol.Any("hours", "minutes", "seconds")): cv.positive_int,
         vol.Optional("name"): cv.string,
+        vol.Optional(
+            "conversation_command",
+            description="Text of command to execute with conversation agent when the timer is finished",
```

Comment:
Home Assistant repurposes this description field to store dictionaries 😬

Reply:
:'(